crypto/hash/hmac_ossl.c

```diff
@@ -225,34 +233,28 @@ static const uint8_t srtp_hmac_test_case_0_tag[SHA1_DIGEST_SIZE] = {
     0xe2, 0x8b, 0xc0, 0xb6, 0xfb, 0x37, 0x8c, 0x8e,
     0xf1, 0x46, 0xbe, 0x00
 };
+// clang-format on
 
 static const srtp_auth_test_case_t srtp_hmac_test_case_0 = {
-    sizeof(srtp_hmac_test_case_0_key),    /* octets in key            */
-    srtp_hmac_test_case_0_key,            /* key                      */
-    sizeof(srtp_hmac_test_case_0_data),   /* octets in data           */
-    srtp_hmac_test_case_0_data,           /* data                     */
-    sizeof(srtp_hmac_test_case_0_tag),    /* octets in tag            */
-    srtp_hmac_test_case_0_tag,            /* tag                      */
-    NULL                             /* pointer to next testcase */
+    sizeof(srtp_hmac_test_case_0_key),  /* octets in key            */
+    srtp_hmac_test_case_0_key,          /* key                      */
+    sizeof(srtp_hmac_test_case_0_data), /* octets in data           */
+    srtp_hmac_test_case_0_data,         /* data                     */
+    sizeof(srtp_hmac_test_case_0_tag),  /* octets in tag            */
+    srtp_hmac_test_case_0_tag,          /* tag                      */
+    NULL                                /* pointer to next testcase */
 };
 
 /* end test case 0 */
 
-static const char srtp_hmac_description[] = "hmac sha-1 authentication function";
+static const char srtp_hmac_description[] =
+    "hmac sha-1 authentication function";
 
 /*
  * srtp_auth_type_t hmac is the hmac metaobject
  */
 
-const srtp_auth_type_t srtp_hmac  = {
-    srtp_hmac_alloc,
-    srtp_hmac_dealloc,
-    srtp_hmac_init,
-    srtp_hmac_compute,
-    srtp_hmac_update,
-    srtp_hmac_start,
-    srtp_hmac_description,
-    &srtp_hmac_test_case_0,
-    SRTP_HMAC_SHA1
-};
-
+const srtp_auth_type_t srtp_hmac = {
+    srtp_hmac_alloc,       srtp_hmac_dealloc,      srtp_hmac_init,
+    srtp_hmac_compute,     srtp_hmac_update,       srtp_hmac_start,
+    srtp_hmac_description, &srtp_hmac_test_case_0, SRTP_HMAC_SHA1};
```

Comment:
why does it need to change from the old initialization structure, no rules where broken and I think it made more sense.